doc/etc/config.ini

```diff
@@ -25,9 +25,9 @@
 # unix_socket_path      The path of the UNIX domain server socket.
 
 [Server]
-address                 = # address
-port                    = # port for the incoming connections on legacy protocol
-neoport                 = # port for the incoming connections on the neo protocol
+address                 = 0.0.0.0
+port                    = 10000
+neoport                 = 10001
```

Comment:
Maybe keep the comments as pure comments above each value? (the options are pretty self explanatory though, so is fine if you want to remove them).

Reply:
I'll remove them here as they are still in the file above the section.